test/helpers/kubectl.go

```diff
@@ -332,6 +332,37 @@ func (kub *Kubectl) GetNumCiliumNodes() int {
 	return len(strings.Split(res.SingleOut(), " ")) - sub
 }
 
+// CountMissedTailCalls returns the number of the sum of all drops due to
+// missed tail calls that happened on all Cilium-managed nodes.
+func (kub *Kubectl) CountMissedTailCalls() (int, error) {
+	ciliumPods, err := kub.GetCiliumPods(GetCiliumNamespace(GetCurrentIntegration()))
+	if err != nil {
+		return -1, err
+	}
+
+	totalMissedTailCalls := 0
+	for _, ciliumPod := range ciliumPods {
+		cmd := "cilium metrics list -o json | jq '.[] | select( .name == \"cilium_drop_count_total\" and .labels.reason == \"Missed tail call\" ).value'"
+		res := kub.CiliumExecContext(context.Background(), ciliumPod, cmd)
+		if !res.WasSuccessful() {
+			return -1, fmt.Errorf("Failed to run %s in pod %s: %s", cmd, ciliumPod, res.CombineOutput())
+		}
+		if res.Stdout() == "" {
```

Comment:
That won't compile on the v1.7 branch. You can use `res.GetStdOut()` instead IIRC.

In general, I found using `git rebase --exec 'make clean && make -j$(nproc)' v1.7` to be useful to check each commit compiles correctly and quickly find which break otherwise (also work with PR against master).

Reply:
Thanks for the quick tip :rocket:. I was using `make build` on each step to check if the commit builds successfully.
Will check this out though, thanks again.

Reply:
Ah, that doesn't build the tests I think. You need:
```
git rebase --exec 'make clean && make -j$(nproc) && cd test && make -j$(nproc)' v1.7
```